src/apiState.ts

```diff
@@ -28,6 +29,7 @@ export type MutationKeys<Definitions extends EndpointDefinitions> = {
 }[keyof Definitions];
 
 type BaseQuerySubState<D extends BaseEndpointDefinition<any, any, any>> = {
+  originalArgs: QueryArgFrom<D>;
```

Comment:
This looks good to me, and I think is probably a good name to stick with. As some possible alternatives for thought: `functionArgs`, `thunkArgs`, `userArgs`, or `givenArgs`.